zenodo/modules/profiles/templates/zenodo_profiles/search.html

```diff
@@ -0,0 +1,55 @@
+{#-
+# This file is part of Zenodo.
+# Copyright (C) 2017 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+-#}
+
+{% extends config.SEARCH_UI_SEARCH_TEMPLATE %}
+
+{%- from "zenodo_profiles/macros.html" import owner_header %}
+
+{% set search_hidden_params = {"q": "owners:" + user.id|string } %}
+
+{%- block page_header %}
+{% set search_input_placeholder = "Search " + user.profile.username %}
+{% include "invenio_search_ui/header.html" %}
+{%- endblock page_header %}
+
+{% block page_body %}
+{%- set global_vars = {'orcid_id': ''} %}
```

Comment:
It seems strange that this is not available in this scope, since it's passed from the template. Can you take a look and check if it's possible to remove usage of a  `global_vars` helper variable?